src/wrappers/React.js

```diff
@@ -46,7 +44,8 @@ const makeReactContainer = Component => {
 export default {
   props: ['component', 'passedProps'],
   render (createElement) {
-    return createElement('div', { ref: 'react' })
+    this.refString = `react-${Math.random().toString()}`
```

Comment:
I don't think doing this on every `render` call is a good idea. Perhaps we could use a `beforeCreated` hook?